crates/sqllib/src/casts.rs

```diff
@@ -14,31 +19,113 @@ use num::{FromPrimitive, One, ToPrimitive, Zero};
 use num_traits::cast::NumCast;
 use regex::{Captures, Regex};
 use rust_decimal::{Decimal, RoundingStrategy};
-use std::collections::BTreeMap;
-use std::error::Error;
-use std::string::String;
+use std::{cmp::Ordering, collections::BTreeMap, error::Error, string::String};
+
+#[doc(hidden)]
+pub(crate) fn type_name(name: &'static str) -> &'static str {
+    match name {
+        "i8" => "TINYINT",
+        "i16" => "SHORTINT",
+        "i32" => "INTEGER",
+        "i64" => "BIGINT",
+        "f" => "REAL",
+        "d" => "FLOAT",
+        "Timestamp" => "Timestamp",
+        "Date" => "DATE",
+        "Time" => "TIME",
+        "decimal" => "DECIMAL",
+        "ShortInterval" => "INTERVAL",
+        "LongInterval" => "INTERVAL",
+        "s" => "(VAR)CHAR",
+        _ => "Unexpected type",
+    }
+}
+
+#[doc(hidden)]
+pub(crate) fn ttype_name(name: &'static str) -> &'static str {
```

Comment:
This seems to be the same function as the one above? or maybe I'm missing smth

Reply:
I must be missing something

Reply:
there are differences, like "d"/"F64"